lib/classes/Variables.js

```diff
@@ -650,14 +650,19 @@ class Variables {
   }
 
   getValueFromCf(variableString) {
+    const regionSuffix = variableString.split(':')[0].split('.')[1];
     const variableStringWithoutSource = variableString.split(':')[1].split('.');
     const stackName = variableStringWithoutSource[0];
     const outputLogicalId = variableStringWithoutSource[1];
+    const options = { useCache: true };
+    if (regionSuffix !== undefined) {
```

Comment:
Please use _.isUndefined of lodash for consistancy

Reply:
Updated in https://github.com/serverless/serverless/pull/5579/commits/13de2a9083d2a04b3c8f29d816a188fea316320e